lib/contracts/index.ts

```diff
@@ -65,4 +66,17 @@ export const contracts: ContractDefinition[] = [
 	viewAllTransformers,
 	viewAllViews,
 	viewScheduledActions,
+
+	relationshipAnyIsCreatorOfAny,
```

Comment:
Was it decided to go ahead and merge in this relationship for the reaction work?